Mathlib/Data/NNReal/Basic.lean

```diff
@@ -336,6 +336,8 @@ noncomputable example : LinearOrder ℝ≥0 := by infer_instance
 
 @[bound] private alias ⟨_, Bound.coe_pos_of_pos⟩ := coe_pos
 
+lemma coe_pos_iff_ne_zero : (0 : ℝ) < r ↔ r ≠ 0 := coe_pos.trans pos_iff_ne_zero
```

Comment:
Please don't mix the `<` and `≠` spellings. Either prove `(0 : ℝ) < r ↔ 0 < r` or `r ≠ (0 : ℝ) ↔ r ≠ 0`, but not something in between